pkg/runner/cluster_k8s.go

```diff
@@ -753,6 +753,10 @@ func (c *ClusterK8sRunner) createTestplanPod(ctx context.Context, podName string
 							Name:          "pprof",
 							ContainerPort: 6060,
 						},
+						{
+							Name:          "lotus-rpc",
+							ContainerPort: 1234,
+						},
```

Comment:
Maybe this should be part of the `manifest.toml` where test plan can define what ports they want exposed on the test plan containers / pods. @raulk WDYT?

Reply:
I added `exposed_ports` to the `DockerfileExtensions`, which get picked up automatically by `local:docker`, as we are also using `PublishAllPorts`. 

I could iterate on them here as well, and map `pod ports` to the container in k8s.